### DISCUSSION TITLE
chaos-daemon pods do not start when installed via Helm using git-bash on Windows 10

### DISCUSSION THREAD

**Discussion by arodindev** [chaos-mesh/chaos-mesh#3954] [Q&A]
## Bug Report

**What version of Kubernetes are you using?**
Client Version: v1.23.12
Server Version: v1.23.15

**What version of Chaos Mesh are you using?**
v2.5.1

**What did you do? / Minimal Reproducible Example**
I installed Chaos-Mesh via Helm as described in the documentation. On my Windows machine i use git-bash. The container runtime on the nodes is containerd, so i used the following installation command:

```
helm install chaos-mesh chaos-mesh/chaos-mesh --namespace=chaos-testing --set chaosDaemon.runtime=containerd --set chaosDaemon.socketPath=/run/containerd/containerd.sock --set dashboard.securityMode=false --version 2.5.1
```

**What did you expect to see?**
I expected to see all pods running.

**What did you see instead?**
The chaos-demon pods failed to start (CrashLoopBackoff), because of incorrect ``chaosDaemon.socketPath``. The pods throw the following error:

```
OCI runtime create failed: container_linux.go:380: starting container process caused: process_linux.go:545: container init caused: rootfs_linux.go:75: mounting "/run/containerd/io.containerd.runtime.v2.task/k8s.io/chaos-daemon/C:/cmder/vendor/git-for-windows/run/containerd" to rootfs at "/host-run" caused: stat /run/containerd/io.containerd.runtime.v2.task/k8s.io/chaos-daemon/C:/cmder/vendor/git-for-windows/run/containerd: no such file or directory: unknown
```

It looks like that, when using git-bash under Windows, the local path of my git-bash installation is added to the ``chaosDaemon.socketPath``. When i use the standard Windows 10 command prompt, everything works fine. I'm not sure if this is a bug related to Chaos-Mesh though.

**Comment by STRRL** [comment]
It seems chaos daemon failed to connect the CRI.

Could you provide more information about your kubernetes cluster? Which platform did it run on? Which CRI did it use?

**Reply by arodindev** [reply]
Running on Azure, CRI is containerd. Everything works fine when using the Azure Cloud Shell or the local Windows 10 Command Prompt. The error only seem to appear when using Git-Bash under Windows 10.

**Reply by AndrewCi** [reply]
I had this same issue. Was resolved when using the command prompt

**Reply by kondasuman** [reply]
> I had this same issue. Was resolved when using the command prompt

running installation from command prompt, will solve this error
